web/flat-web/src/utils/CoursewarePreloader/index.ts

```diff
@@ -0,0 +1,78 @@
+import { BlobReader, BlobWriter, ZipReader } from "@zip.js/zip.js";
+import { cacheMember, contentType } from "./utils";
+
+type PPTInfo = {
+    pptType: "static" | "dynamic";
+    taskUUID: string;
+};
+
+class CoursewarePreloader {
+    public downloaders = new Map<string, Response>();
+
+    public async preload(pptSrc: string): Promise<void> {
+        const pptInfo = CoursewarePreloader.parsePPTURLInfo(pptSrc);
+        const { pptType, taskUUID } = pptInfo;
+
+        if (this.downloaders.has(taskUUID)) {
+            return;
+        }
+
+        const zipUrl = `https://convertcdn.netless.link/${pptType}Convert/${taskUUID}.zip`;
+
+        const res = await fetch(zipUrl, {
```

Comment:
```ts
const res = await fetch("https://convertcdn.netless.link/${pptType}Convert/${taskUUID}.zip", {
    method: "get",
});
```

why not use Axios

Reply:
Why use Axios instead of the natively supported fetch method?

Reply:
Because axios we have encapsulated the retry mechanism to facilitate our unified management

Reply:
I would love it if we had a place to manage `fetch`, but at the moment we don't, we only have `axios`

Reply:
ok

Reply:
it seems UploadTask.ts already give `getNotAuth` method. Maybe should use that methods.